src/ngx_http_upsync_module.c

```diff
@@ -461,22 +463,57 @@ ngx_http_upsync_server(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
 
     ngx_memzero(&u, sizeof(ngx_url_t));
 
-    p = (u_char *)ngx_strchr(value[1].data, '/');
+    p = (u_char *)ngx_strchr(value[1].data, '@');
+    if (p == NULL) {
+        s.data = value[1].data;
+        s.len = value[1].len;
```

Comment:
I mean we only need care if p is not null and extracting useful message. If p is null, we shoud be doing nothing, right?